src/helpers/slack.ts

```diff
@@ -1,5 +1,5 @@
 const { Reaction, ErrorCode } = require("@slack/web-api");
-import * as crypto from "crypto";
+const crypto = require("crypto");
```

Comment:
I agree that `crypto` might not be needed and this can be solved by using `Math.random()` and I was even using it as seen here. SonarCloud however ran code quality and security checks and highlighted that the use of `Math.random()` was not secure. Hence I moved it to use `crypto`.

Hopefully that makes sense!

Reply:
I have updated the import though 👍

Reply:
I can understand. This is the issue with tools, they do not think analytically. it should be fine, but if you want you can suppress those using `#pragma warning disable SCODE`